app/views/courses/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: course) do |form| %>
+  <% if course.errors.any? %>
```

Comment:
I know this is boilerplate but we are 100% not going to be keeping this so I would just remove this entire file.